Terminal.Gui/Core/ConsoleDriver.cs

```diff
@@ -458,6 +458,20 @@ static void SetColorScheme (ColorScheme colorScheme, [CallerMemberName] string c
 		public static Dictionary<string, ColorScheme> ColorSchemes { get; }
 	}
 
+	/// <summary>
+	/// The visible height should be used on the window.
+	/// </summary>
+	public enum HeightSize {
```

Comment:
I'm having trouble understanding the naming here. `HeightSize` doesn't see right. Would a more descriptive name be 'TerminalHeight` or `ConsoleHeight`? Or `VisibleHeight` (since the doc reads `the *visible height*`)?

Actually...

Why is this an enum and not just a boolean? Do you believe you'll have more than 2 values in the future (behind Window and Buffer)?

Why not simplify this by making this 

```
/// if false height is measured by the window height and thus no scrolling. If true then height is measured by the buffer height, enabling scrolling
public bool HeightAsBuffer {get;set;}
```

?

Or, why not name it around the scrolling behavior (because I think that's what matters)?

Reply:
I think `public bool HeightAsBuffer {get;set;}` is fine to me.